src/test/java/petStoreTests/user/AddUserTest.java

```diff
@@ -0,0 +1,31 @@
+package petStoreTests.user;
+
+import client.UserClient;
+import data.DataSet;
+import dto.requests.ResponseInfo;
+import io.qameta.allure.Description;
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import petStoreTests.AbstractTest;
+
+import java.io.IOException;
+
+public class AddUserTest extends AbstractTest {
+
+    @Description("Verify that we are adding user to the store data base")
+    @Test
+    public void addUserTest() throws IOException {
+        LOGGER.info("START TEST add user to the store data base");
+        ResponseInfo response = UserClient.postUser(DataSet.addingUser());
+
+        Assert.assertEquals(response.getCode(), HttpStatus.SC_OK);
+        Assert.assertEquals(response.getType(), DataSet.messageUnknownResponse().getType());
+        Assert.assertEquals(response.getMessage(), String.valueOf(DataSet.addingUser().getId()));
```

Comment:
Also add GET request for the specified user by id and verify that returned user is equal to the one you posted